### ISSUE TITLE
Sails bug http req/min

### ISSUE THREAD

**Issue by Unitech** [keymetrics/pm2-io-apm#147]
Dear Support,

I'm running a node (sails.js) app with pm2 and after the update to version 3.0.0 it doesn't show me correct "http requests/min" statistics. It's always 0. The counter for an express website on the same instance seems to work correctly. Do you have any solution or hints for this problem?

kind regards

**Comment by camparound-developers** [comment]
Here are some more information:
I don't use pm2-io-apm (as far as i know and how i understand it), only pm2 as process manager for an express.js and a sails.js server. The counter for the express server delivers correct data, the counter for sails does not (always 0) since i updated pm2 to version 3.0.0. It worked fine with pm2 v2.x (i don't remember the subversion).

**Comment by Unitech** [comment]
Thanks for this complementary information.
Which sailsjs version do you use?

**Comment by camparound-developers** [comment]
I'm using sails v0.12.13

**Comment by Unitech** [comment]
Bug confirmed on latest Sails

Reproduce:

```bash
$ npm install sails -g
$ sails new sails-project
$ cd sails-project
$ pm2 start app.js
$ pm2 monit
```

By doing queries, the http req/min + pmx:http:latency does not change 

Express hook:
https://github.com/keymetrics/pm2-io-apm/blob/master/src/metrics/network.ts#L136

**Comment by Unitech** [comment]
bug: Double pmx:http:latency probe instanciation

**Comment by wallet77** [comment]
It will be shipped in version 2.1.0.
And it is already available on version 2.1.0-beta4
